src/v0/destinations/google_adwords_offline_conversions/data/storeAddoperationAddressMap.json

```diff
@@ -0,0 +1,67 @@
+[
+    {
+        "sourceKeys": "city",
+        "destKey": "city",
+        "sourceFromGenericMap": true
+    },
+    {
+        "sourceKeys": "state",
+        "destKey": "state",
+        "sourceFromGenericMap": true
+    },
+    {
+        "sourceKeys": [
+            "context.traits.country_code",
+            "properties.country_code",
+            "context.traits.countryCode",
+            "properties.countryCode"
+        ],
+        "destKey": "country_code"
+    },
+    {
+        "sourceKeys": [
+            "context.traits.postal_code",
+            "properties.postal_code",
+            "context.traits.postalCode",
+            "properties.postalCode",
+            "context.traits.zipCode",
+            "properties.zipCode"
+        ],
+        "destKey": "postal_code"
+    },
+    {
+        "sourceKeys": "firstName",
+        "destKey": "hashed_first_name",
+        "sourceFromGenericMap": true,
+        "metadata": {
+            "type": [
+                "toLower"
+            ]
+        }
+    },
+    {
+        "destKey": "hashed_street_address",
+        "sourceKeys": [
+            "properties.streetAddress",
+            "context.traits.streetAddress",
+            "traits.street",
+            "context.traits.street"
+        ],
+        "metadata": {
+            "type": [
+                "toLower"
+            ]
+        }
+    },
+    {
+        "sourceKeys": "lastName",
+        "destKey": "hashed_last_name",
+        "metadata": {
+            "type": [
+                "toLower",
+                "hashToSha256"
+            ]
+        },
+        "sourceFromGenericMap": true
+    }
+]
```

Comment:
We have to add `hashToSha256` for all the fields where we need to hash.

Reply:
Yeah we have changed it. Now we are hashing it depending on the `hasUserIdentifier` Flag as we are doing it for email and phone